### PR TITLE
Deal better with datasets that use `\r\n` as line separator

### LINKED ISSUE #81: OpusCleaner leaves `\r` in output?
Nick ran into some issues with spurious `\r` carriage return characters in the output files. That should not happen.

### PR DESCRIPTION
**Pull request by jelmervdl**
Fixes #81 .

### CODE REVIEW

File: opuscleaner/filters/fix_un_chinese.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
 
 for line in sys.stdin:
-    line = line[:-1] #EoL
     line = line.strip()
```
Comment (XapaJIaMnu) [inline opuscleaner/filters/fix_un_chinese.py:11]: should this be `rstrip('\r\n')`

**Review by XapaJIaMnu — APPROVED** [review]
Just one minor suggestion.

File: opuscleaner/filters/fix_un_chinese.py (hunk 1 above)
Comment (jelmervdl) [reply opuscleaner/filters/fix_un_chinese.py:11]: I don't think that makes sense, everything `rstrip(\r\n)` would remove would also get nuked by strip() on the following line anyway.

Comment (XapaJIaMnu) [reply opuscleaner/filters/fix_un_chinese.py:11]: Ok, having read the documentation, i get it now.

**Pull request merged**